operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml-patch

```diff
@@ -17,3 +17,4 @@
       - csi.sharedresource.openshift.io
       - diskplugin.csi.alibabacloud.com
       - vpc.block.csi.ibm.io
+      - csi.nutanix.com
```

Comment:
Please add this to the `operator/v1/0000_90_cluster_csi_driver_01_config.crd.yaml` file as well

Reply:
ok @bertinatto i just add it to in the file with [7ee0d1f](https://github.com/openshift/api/pull/1138/commits/7ee0d1fe8892b8825df5353ca342842d490fe587)
but it was not specified in the doc you share with us , i imagined it was automatically generated base on the patch file